zipline/testing/fixtures.py

```diff
@@ -224,18 +272,36 @@ class WithAssetFinder(object):
     zipline.testing.make_future_info
     zipline.testing.make_commodity_future_info
     """
+    ASSET_FINDER_EQUITY_SIDS = tuple(map(ord, 'ABC'))
```

Comment:
This is substantially less clear than just `ord('A'), ord('B'), ord('C')`.  In particular, it's not obvious at a glance that there are 3 sids here.